packages/flutter/lib/src/services/message_codecs.dart

```diff
@@ -245,20 +245,20 @@ class StandardMessageCodec implements MessageCodec<dynamic> {
   // * Maps are encoded by first encoding their length in the expanding format,
   //   then follows the recursive encoding of each key/value pair, including the
   //   type byte for both (Maps are assumed to be heterogeneous).
-  static const int _kNull = 0;
-  static const int _kTrue = 1;
-  static const int _kFalse = 2;
-  static const int _kInt32 = 3;
-  static const int _kInt64 = 4;
-  static const int _kLargeInt = 5;
-  static const int _kFloat64 = 6;
-  static const int _kString = 7;
-  static const int _kUint8List = 8;
-  static const int _kInt32List = 9;
-  static const int _kInt64List = 10;
-  static const int _kFloat64List = 11;
-  static const int _kList = 12;
-  static const int _kMap = 13;
+  static const int _null = 0;
+  static const int _true = 1;
+  static const int _false = 2;
+  static const int _int32 = 3;
+  static const int _int64 = 4;
+  static const int _largeInt = 5;
+  static const int _float64 = 6;
+  static const int _string = 7;
+  static const int _uint8List = 8;
+  static const int _int32List = 9;
+  static const int _int64List = 10;
+  static const int _float64List = 11;
+  static const int _list = 12;
+  static const int _map = 13;
```

Comment:
for things like this I think we should have a common prefix or suffix to indicate that they are related.

Reply:
Done. (named them all _valueFoo)